content/en-us/designs/protocol-translation/_index.md

```diff
@@ -0,0 +1,198 @@
+* Connecting the devices directly to the internet would create a security risk. This often arises because the local protocol does not support encryption or authentication mechanisms.
+
+## Solution
+IoT solutions address these challenges by applying the [Gateway]({{< ref "/designs/gateway" >}}) in combination with edge logic running on the gateway.
+The edge logic on the gateway executes the translation of the industrial protocols into a common data format/protocol that can be processed by backend IT systems such as databases, data lakes or webservices.
+
+The Protocol Translation design shown in the following diagram can be used to solve these challenges.
+
+![Protocol Translation Design](ProtocolTranslation.png)([PPTx](/designs/iot-atlas-patterns.pptx))
+
+- The left portion of this diagram represents the devices at the edge of the IoT solution that are connected with at least one [Gateway]({{< ref "/designs/gateway" >}}).
+- The gateway is the only component connected to the internet and the red arrow indicates that this connection should be encrypted/protected accordingly.
+- The gateway runs local protocol translation logic to translate different protocols into a common protocol.
+- If data generated by one of the connected devices has to be pushed to the cloud, the gateway will securely transmit the data in a common format after translation has occurred.
+Since the Protocol Translation design extends the Gateway design, the solution may also allow a device **`B`** to act upon data generated by device **`A`** or vice versa. In this case, the gateway would have protocol translation logic to translate data emitted by device **`A`** into a format for use by the cloud and also protocol translation logic to translate data destined for the cloud into the format usable by device **`B`**.
+ 
+The Protocol Translation design also fills a security purpose:
+The connections inside the local network may be insecure (black) because some of the used protocols have no security features, while the cloud-facing connections (red) are secured by contemporary encryption and authentication mechanisms.
+While encrypting all connections is desirable, in many cases this would prevent using well-established protocols for certain applications that do not support such security mechanisms. Furthermore, for certain devices and applications, replacing devices that do not support contemporary security mechanisms is not economically feasible. Often this means using insecure protocols locally in a solution is an acceptable tradeoff.
+
+## Considerations
+
+When implementing this design, consider the following questions:
+
+#### Does the schema for the translation logic change if the connection is not IP-based?
+In general, the schema does not change.
+In the example [below](#example---industrial-iot-data-extraction), the IP connection was always defined using an IP address.
```

Comment:
Link should be `#example---industrial-iot-data-extraction` (only single dash needed)